upip/upip.py

```diff
@@ -268,7 +381,7 @@ def main():
         help()
         return
 
-    if sys.argv[1] != "install":
+    if sys.argv[1] not in ["install"]:
```

Comment:
I think the original is more readable. The 'not in' form is great for testing if an item is in a collection but that's not the case here.